test/test_autograd.py

```diff
@@ -3965,6 +3955,131 @@ def backward(ctx, grad_out):
         gradcheck(lambda x: NonDetFunc.apply(x, 1e-6), inp, nondet_tol=1e-5, check_batched_grad=False)
         gradgradcheck(lambda x: NonDetFunc.apply(x, 1e-12), inp, nondet_tol=1e-5, check_batched_grad=False)
 
+    def test_gradcheck_test_inputs(self):
```

Comment:
nit: "validates inputs" or "validates test inputs"  is more descriptive than "test inputs"